### ISSUE TITLE
Issue in "Plain GD" model at end of document "Zalando"

### ISSUE THREAD

**Issue by heimschf** [Apress/applied-deep-learning#2]
Using your awesome book and your github repository, I try to cover the very nicely elaborated neural networks. 

Perhaps this is why I am rather a newbie to python: But when I assess the predicted variables after your code in In 17:

```
correct_predictions = tf.equal(tf.argmax(y_,0), tf.argmax(Y,0))
accuracy = tf.reduce_mean(tf.cast(correct_predictions, "float"))
print ("Accuracy:", accuracy.eval({X: test, Y: labels_test_, learning_rate: 0.001}, session = sess))
prediction = y_
mypreds = (prediction.eval(feed_dict={X: train}, session = sess))
cs = np.sum(mypreds,axis=0)
rs = np.sum(mypreds,axis=1)
print(cs)
print(rs)
```

I actually want to test whether the normalization of the outputs has properly worked, e.g. that probabilities among the 10 items 0-9 add up to 1. For all models before (I tested two of them), this works, but for the present case, I see that ROW sums are 1 and not column sums. Perhaps it is a very minor issue or I missed something. 

Kind regards

**Comment by michelucci** [comment]
Hi, thanks for reading the book. I am travelling at a conference until mid next week. If you have patience I will check your questions as soon as I can, but it may take a few days. I will get back to you. ;-) Umberto
